RELEASE_PLAN.md

```diff
@@ -0,0 +1,62 @@
+# RELEASE PLAN
+
+## Acceptance Testing
+
+#### Unit Tests
+
+Run the unit tests against the compiled version of the agent:
+
+```
+TEST_BUILD_ARTIFACT=1 bundle exec rake
+```
+
+#### Kitchen Acceptance Tests
+
+Run the full kitchen acceptance suite
+
+```
+TBD
```

Comment:
After https://github.com/chef/automate-liveness-agent/pull/13 is merged you can `rake compile_recipe && kitchen test` to verify it.